docs/user_guide/quickstart.rst

```diff
@@ -1,158 +1,135 @@
-   macros              ---          ---            0             0             0             0             0             0            ---          ---
-   cells               ---           24            58            58            58            61            61            61           ---          ---
-   registers           ---           9             9             9             9             9             9             9            ---          ---
-   buffers             ---          ---            1             1             1             4             4             4            ---          ---
-   pins                ---          ---            3             3             3             3             3             3            ---          ---
-   nets                ---          ---            37            37            37            40            40            40           ---          ---
-   vias                ---          ---           ---           ---           ---           ---           185           ---           ---          ---
-   wirelength          ---          ---           ---           ---           ---           ---          212.0          ---           ---          ---
-   memory          27222016.0   30126080.0    95330304.0    89309184.0   127594496.0   102363136.0   708177920.0    97525760.0   395513856.0   81055744.0
-   exetime             0.21         0.51          0.41          0.31          0.51          1.42          2.36          0.31          1.24         0.41
-   tasktime            0.35         1.4           0.83          0.6           0.8           1.71          2.69          0.66          2.2          0.74
-
-
-If you find that having all the tool-specific information printed to the screen is too noisy, you can set the :keypath:`option,quiet` parameter to True.
+Once you have verified that your remote configuration works, try compiling a simple design:
 
 .. code-block:: bash
 
-   chip.set('option','quiet',True)
-   
+   (venv) sc -target asic_demo -remote
 
-Even if you use the :ref:`quiet option <quiet>`, you will still be able to access the tool-specific output from the log file that is saved for each tool. You can find the log files associated with each tool in: ``build/<design>/<jobname>/<step>/<index>/<step>.log``
 
+.. _remote run results:
 
-By default, all SiliconCompiler outputs are placed in the ``build/<design>`` directory.
+Remote Run Results
+^^^^^^^^^^^^^^^^^^
 
-.. note::
+.. include:: quickstart/quickstart_banner.rst
+
+As run goes through each step of the flow, a message will be printed to the screen every 30 seconds.
+
+Then, at the end of the run, a summary table will be printed similar to the one show below:
+
+.. image:: ../_images/summary_table.png
+
+All design outputs are located in ``build/<design>/<jobname>``, and you will be able to find a report with a screenshot of the demo design and the summary table.
```

Comment:
re: "All design outputs", the public server is only sending back the manifest and reports.

Reply:
Updated to include difference between remote and local in outputs.